packages/ethers5/src/client.ts

```diff
@@ -11,9 +12,10 @@ import type {
   Token
 } from '@web3modal/scaffold'
 import { Web3ModalScaffold } from '@web3modal/scaffold'
-import type { Web3ModalSIWEClient } from '@web3modal/siwe'
+import { SIWEController, type Web3ModalSIWEClient } from '@web3modal/siwe'
```

Comment:
SIWEController should be dynamically imported
```suggestion
import type { Web3ModalSIWEClient } from '@web3modal/siwe'
```